### DISCUSSION TITLE
Version 1.1b2-Cuda for windows flagged as virus

### DISCUSSION THREAD

**Discussion by cacalo** [CarlGao4/Demucs-Gui#51] [Q&A] [answered]
Hi there!
I just tried downlading the latest version of the GUI, (Demucs-GUI_1.1b2_cuda.7z) and Google Chrome is flagging the file as virus and removing it.
Is someone else having the same issue?
<img width="242" alt="Captura de pantalla_20230201_184256" src="https://github.com/CarlGao4/Demucs-Gui/assets/60068962/d5d4f7c0-3035-41af-9fee-4e2ca09bba89">

**Comment by CarlGao4** [comment, marked as answer]
I’ve explained it in [usage.md](https://github.com/CarlGao4/Demucs-Gui/blob/main/usage.md#the-application-triggers-my-antivirus-software)

**Reply by CarlGao4** [reply]
And I attached a report of VirusTotal when releasing [1.0](https://www.virustotal.com/gui/file/503f39f7b2611b99daeadde1946647bb596423414542cd41f4f9f3a45bce20f6/detection)

**Reply by cacalo** [reply]
I'm sorry for being such stupid. It seemed odd that older versions didn't trigger the browser's anti virus check but after reading your docs I now understand why.
Thank you for the clarification.
